examples/inspection/plot_linear_model_coefficient_interpretation.py

```diff
@@ -289,11 +290,12 @@
 # %%
 # Now that the coefficients have been scaled, we can safely compare them.
 #
-# .. warning::
+# .. note::
```

Comment:
The term warning to me seemed to show that something was wrong in this example.  (and the questions, without answer, seemed to indicate that too).